### PR TITLE
added alpha option to curtain function

### PR DESCRIPTION
**Pull request by orena1**
e.g. 
curtain

### CODE REVIEW

File: stackview/__init__.py [hunk 1]
```diff
@@ -116,6 +116,7 @@ def curtain(
         display_width: int = None,
         display_height: int = None,
         continuous_update: bool = False
+        alpha: float = 1,
```
Comment (haesleinhuepf) [inline stackview/__init__.py:119]:
```suggestion
        continuous_update: bool = False,
        alpha: float = 1
```

**Review by haesleinhuepf — APPROVED** [review]
Amazing @orena1 ! I'm just fixing a tiny typo on your branch and merge afterwards

**Comment by haesleinhuepf** [comment]
Amazing, thanks so much @orena1 ! You can try it out using 

pip install stackview==0.3.6

And I presume it'll be on conda-forge tomorrow as well. Thanks again!

**Comment by orena1** [comment]
Thanks

**Pull request merged**